web/src/app/modules/data/modules/data-service/services/specmate-data.service.ts

```diff
@@ -455,4 +461,36 @@ export class SpecmateDataService {
             await this.readElement(parent, false);
         }
     }
+
+    public async saveModelImage(element: IContainer) {
+        if (Type.is(element, CEGModel) || Type.is(element, Process)) {
+            let model = element as CEGModel | Process;
+            let svg: SVGSVGElement = document.getElementById('mxGraphContainer').getElementsByTagName('svg')[0];
+            let minWidth = svg.style.minWidth;
+            let minHeight = svg.style.minHeight;
+            svg.style.width = '0px';
+            svg.style.height = '0px';
+
```

Comment:
the next lines are full of magic values, please refactor to constants and comment on what is going on